packages/fxa-settings/src/pages/Signup/Confirm/index.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import Confirm, { ConfirmProps } from '.';
+import AppLayout from '../../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { MOCK_ACCOUNT } from '../../../models/mocks';
+import { MOCK_GOBACK_CB } from './mocks';
+
+export default {
+  title: 'pages/Signup/Confirm',
+  component: Confirm,
+} as Meta;
+
+const storyWithProps = ({ ...props }) => {
```

Comment:
Small nit, if you make this `const storyWithProps = (props: ConfirmSigninPropsOrWhateverThisIsNamed) => {`, really just without the object and spread, you can still use it how you are and then make the default story `storyWithProps();` without the empty object.